### ISSUE TITLE
Stop spamming

### ISSUE THREAD

**Issue by bobince** [hola/challenge_haggling#3]
challengejs@hola.org is promoting this competition by sending unsolicited e-mail to addresses without any permission from the victim and, contrary to CAN-SPAM, with no identification or unsubscribe link.

This is unacceptable. STOP.

**Comment by keegancsmith** [comment]
Yeah, thanks for sending me 3 copies of the same SPAM I never signed up for.
